.github/workflows/test.yml

```diff
@@ -16,8 +16,10 @@ jobs:
     - name: Run bash test scripts
       shell: bash
       run: |
-        chmod +x ./test.sh
-        ./test.sh
+        git config --global user.name "GitHub Actions"
+        git config --global user.email "nobody@localhost"
+        chmod +x ./test-all.sh
```

Comment:
The exec-bit is set in the repository. Is it really necessary?

Reply:
would `bash test.sh` be more precise then?

Reply:
I didn't actually add the `chmod +x` line, as it was already there. (https://github.com/eficode-academy/git-katas/pull/299/files/95d4a5eacc0f58dc1df707ffe40d84881fb5edab)
I was aware that the bit is already set in the repo, but I assumed that the `chmod +x` had been added at some point "for safety", and it does make sure that the pipeline still works even if someone on e.g. windows manages to commit without the executable bit.
I guess it doesn't hurt to have this, so I see no reason not to leave it in there.

@zanderhavgaard I have no strong opinions, but I believe that the `./test-all.sh` form, running with the shell defined in the shebang in the file, is the most idiomatic way of running scripts, as it relies on the shel that the script author has declared the script to work with (and by extension also the language spec that shellcheck will lint the script against if you use that).